HTML/Logic/skin.css

```diff
@@ -68,3 +68,9 @@ div#background {
 	background-image: url('html/images/btn_settings.gif');
 }
 
+.settings .buttonbar {
+	padding: 6px 13px 0px;
+}
+.x-btn-mr {
```

Comment:
You can't modify `.x-btn-mr`, as this would modify _all_ buttons, not only this one. I think what you want is to add some padding right to `#pluginButtonBar #filterChooser`. I'll do that - please let me know if that wasn't what you were looking for. Thanks!